@xen-orchestra/lite/src/stories/web-core/toaster/ui-toaster.story.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <ComponentStory
+    v-slot="{ properties, settings }"
+    :params="[
+      prop('accent')
+        .type('ToasterColor')
```

Comment:
To match the real type in the component:
```suggestion
        .type('ToasterAccent')
```